src/include/main/settings.h

```diff
@@ -34,10 +35,15 @@ struct WarningLimitSetting {
 
 struct TimeoutSetting {
     static constexpr auto name = "timeout";
-    static constexpr auto inputType = common::LogicalTypeID::INT64;
+    static constexpr auto inputType = common::LogicalTypeID::DOUBLE;
```

Comment:
We can make timeout value also UINT64.
The timeout value stored in clientConfig is uint64_t:
```
struct ClientConfigDefault {
    // 0 means timeout is disabled by default.
    static constexpr uint64_t TIMEOUT_IN_MS = 0;
```
it doesn't make sense to support timeout value as double